GCM/Users/users/User.java

```diff
@@ -57,45 +47,9 @@ public String setUsername(String username) {
 		return this.username = username;
 	}
 
-	public Date getMembershipExpireDate() {
```

Comment:
Why is this deleted

Reply:
Purchase history will be stored in separate class